mmdet/models/necks/fpn.py

```diff
@@ -19,7 +19,9 @@ def __init__(self,
                  extra_convs_on_inputs=True,
                  conv_cfg=None,
                  normalize=None,
-                 activation=None):
+                 activation=None,
+                 relu_extra_convs=False,
```

Comment:
May be better to rename it to `relu_before_extra_convs`. This argument can be moved after `extra_convs_on_inputs`.